src/main/app/src/app/zaken/besluit-create/besluit-create.component.ts

```diff
@@ -42,8 +42,14 @@ export class BesluitCreateComponent implements OnInit, OnDestroy {
 
     ngOnInit(): void {
         this.formConfig = new FormConfigBuilder().saveText('actie.aanmaken').cancelText('actie.annuleren').build();
-        const resultaattypeField = new SelectFormFieldBuilder().id('resultaattype').label('resultaat').validators(Validators.required).optionLabel('naam')
-                                                               .options(this.zakenService.listResultaattypes(this.zaak.zaaktype.uuid)).build();
+
+        const heeftBesluit: boolean = this.zaak.besluiten?.length > 0 && !!this.zaak.resultaat;
+        const resultaattypeField = new SelectFormFieldBuilder(heeftBesluit ? this.zaak.resultaat.resultaattype : null).id('resultaattype').label('resultaat')
```

Comment:
Is het nodig om te controleren op besluit, kan je niet altijd het resultaat overnemen?

Dan kan je de code wat simpeler maken;  new SelectFormFieldBuilder(this.zaak.resultaat?.resultaattype)